camp/nfutsum/Twisty/src/main/java/frc/robot/commands/MotorGo.java

```diff
@@ -0,0 +1,54 @@
+
+    // Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import frc.robot.subsystems.NaherNeoMotor;
+import edu.wpi.first.wpilibj.Joystick;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+/** An example command that uses an example subsystem. */
+public class MotorGo extends CommandBase {
+  @SuppressWarnings({"PMD.UnusedPrivateField", "PMD.SingularField"})
```

Comment:
please provide a comment on why to suppress or remove suppression

Reply:
I don't think I needed that line so I removed it